### ISSUE TITLE
ARM64 Container Builds

### ISSUE THREAD

**Issue by hyperschwartz** [provenance-io/provenance#1634]
## Summary

The produced docker containers for this repository currently only support AMD64 architectures.  It would be amazing to get ARM64 multi-platform builds for the containers in the future.

## Problem Definition

Figure does a large amount of testing with locally-built Provenance containers.  With the introduction of the M1 Macbook and later generations, local development has decreased in efficiency due to the need for these machines to do AMD64 emulation when running the containers.

## Proposal

Please modify the docker build/publication to include ARM64 multi-platform images.  Docker buildx is already used, so I believe this should not require a large number of changes to the build configurations.
 
____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by SpicyLemon** [comment]
A bit ago, we removed the ARM64 build because of problems signing it (with Apple). Building and publishing of the related docker container was also removed the since it needed that executable.

Because of the Apple signing requirement (and required hurdles and fees), we probably don't want to publish the mac executable as a release artifact again. But maybe published as a docker container, Mac's won't incessantly complain about it not being signed.
